Sources/SwiftProtobuf/AnyMessageStorage.swift

```diff
@@ -202,7 +205,18 @@ internal class AnyMessageStorage {
       // never inserted.
       break
 
-    case .contentJSON:
+    case .contentJSON(let contentJSON, let options):
+      do {
+        guard let messageType = Google_Protobuf_Any.messageType(forTypeURL: _typeURL) else {
+            throw BinaryEncodingError.anyTranscodeFailure
+        }
+        _ = try unpack(contentJSON: contentJSON,
```

Comment:
should we comment about this doing a full unpack and throwing it away to then have it repeated, but given current flows, there's nothing we can really do about it?

Reply:
Commented.